js/app.js

```diff
@@ -1,2 +1,264 @@
 'use strict';
 
+var hoursOfOperation = ['6am', '7am', '8am', '9am', '10am', '11am', '12pm', '1pm', '2pm', '3pm', '4pm', '5pm', '6pm', '7pm', '8pm'];
+var dailyTotalSales = 0;
+var ulElement;
+var listElement;
+
+var firstAndPike = {
+  location: '1st and Pike',
+  minCustHour: 23,
+  maxCustHour: 65,
+  avgNumPer: 6.3,
+  custPerHour: [],
+  salesPerHour: [],
+  dailyTotal: 0,
+  storeMin: function(num) { //@param num minimum number of customers per hour
+    this.minCustHour = num;
+  },
+  storeMax: function(num) { //@param num maximum number of customers per hour
+    this.maxCustHour = num;
+  },
+  storeAvgPer: function(num) { //@param num avg number purchased per customer
+    this.avgNumPer = num;
+  },
+  getSalesPerHour: function() { //Store number customers per hour
+    for(var i = 0; i < hoursOfOperation.length; i++) {
+      this.custPerHour.push(Math.floor((Math.random() * (this.maxCustHour - this.minCustHour) + this.minCustHour)));
+    }
+  },
+  storeResultsPerHour: function() { // Store number sales per hour and calculate daily total
+    for(var i = 0; i < hoursOfOperation.length; i++) {
+      var numberSold = Math.round(this.custPerHour[i] * this.avgNumPer);
+      this.salesPerHour.push(numberSold);
+      this.dailyTotal += numberSold;
+    }
+    dailyTotalSales += this.dailyTotal;
+  },
+  renderResults: function() {
+    ulElement = document.getElementById('firstAndPike');
+    listElement = document.createElement('ul');
+    listElement.textContent = (this.location);
+    ulElement.appendChild(listElement);
+
+    for(var i = 0; i < hoursOfOperation.length; i++) {
+      listElement= document.createElement('li');
```

Comment:
style nitpick: be consistent in leaving a space between the variable name and the assignment equals sign